packages/mantine/src/styles/Tooltip.module.css

```
@@ -1,3 +1,4 @@
 .tooltip {
     word-break: break-word;
+    padding: var(--mantine-spacing-sm);
```

They want 12px, but we don't have a variable for it. It falls outside our current spacing grid 😢 

```suggestion
    padding: 12px;
```

Let's hardcode it until we define a proper variable for it.